04/README.md

```diff
@@ -43,3 +43,20 @@ hashPassword("mojeBezpieczneHaslo").then((hash: string) => {
 > :no_entry: *Jeśli nie posiadasz materiałów do tego zadania tj. **PDF + wideo, projekt + Code Review**, znajdziesz je na stronie [devmentor.pl](https://devmentor.pl/workshop-ts-typing-advanced)**
 
 > :arrow_left: [*poprzednie zadanie*](./../03) | [*następne zadanie*](./../05) :arrow_right:
+
+
+npm run json-server
+
+npm run dev
+
+1. dlaczego to nie działa
+
+const url = 
```

Comment:
Nie rozumiem do końca, co nie działa. Raczej powinno być ok - pokażesz na spotkaniu.


PS. To zadanie na pewno dało się zrobić dużo prościej. Nie trzeba było instalować Vite lub podobnego bundlera. Jeśli masz rozdzielony kod na pliki to można to zrobić tak, że głównym plik np. `app.js` podpinasz pod HTML przy pomocy `<script type="module" src="./app.js"></script>` (tutaj ważne jest `type="module"` oraz to, że pliki mają wskazane rozszerzenia) i reszta importów powinna działać.